src/components/container/ContainerComponent.tsx

```diff
@@ -46,6 +46,7 @@ interface ContainerProps<TData = unknown> {
   noUnselection?: boolean;
   lockZoom: boolean;
   lockPan: boolean;
+  zoomWithoutAltKey?: boolean;
```

Comment:
Can we think about something more generic? I'm sure we will want to also pan without the Alt key at some point, or use the cmd key instead, etc.

Reply:
Pan without the alt command is always on in the "select" mode since this mode does not allow drawing.

Other modes need the alt key as a way to differentiate pan with draw.

I suggest to rename to `zoomWithoutModifierKey` which will allow us if needed to add `panZoomModifierKey` without a breaking change.

Reply:
ok